conf/scylla.yaml

```diff
@@ -660,3 +660,14 @@ maintenance_socket: ignore
 # Note that creating keyspaces with tablets enabled or disabled is irreversible.
 # The `tablets` option cannot be changed using `ALTER KEYSPACE`.
 enable_tablets: true
+
+# Enforce tablets for new keyspaces.
+# When enabled, newly created keyspaces will always have tablets enabled by default.
```

Comment:
What if replication strategy doesn't support tablets? E.g. SimpleStrategy

Reply:
We need to return an error to create in this case.

Reply:
Added unit tests to verify the error in v2